README.md

```diff
@@ -106,6 +106,23 @@ testing and development purposes.
      ansible/prepare_devel_test_runners.yml
    ```
 
+## User Guide
+
+### Re-running tasks
+
+You can use the re-run label to:
+
+* re-run failing tasks: once the label is added, the tasks with status error
+  or failure will be mark as unassigned, and latter they will be run.
+
+* run the PR using the config file (.freeipa-pr-ci.yaml) of the target branch:
```

Comment:
The wording of this seems confusing... Are you trying to describe the case when new tasks are added to config, but they haven't been created/executed for the PR?